packages/logs/src/boot/startLogs.ts

```diff
@@ -37,10 +39,14 @@ export function startLogs(
   getGlobalContext: () => Context
 ) {
   const { configuration, internalMonitoring } = commonInit(initConfiguration, buildEnv)
-  const errorObservable =
-    initConfiguration.forwardErrorsToLogs !== false
-      ? startAutomaticErrorCollection(configuration)
-      : new Observable<RawError>()
+  const errorObservable = new Observable<RawError>()
+
+  if (initConfiguration.forwardErrorsToLogs !== false) {
+    trackConsoleError(errorObservable)
+    trackRuntimeError(errorObservable)
+    trackNetworkError(configuration, errorObservable, configuration.isEnabled('remove-network-errors'))
```

Comment:
When releasing V3, I don't think we'll remove errors for aborted requests for logs, so this extra parameter may be removed. Nonetheless, we can leave it here until we get confirmation from @hdelaby

Reply:
Yes, I think this decision does not impact this PR since we are still in V2. Maybe it can be added to the v3 release plan when @hdelaby will confirm.